src/Symfony/Component/DependencyInjection/Compiler/Compiler.php

```diff
@@ -95,6 +102,8 @@ public function addPass(CompilerPassInterface $pass, $type = PassConfig::TYPE_BE
      */
     public function addLogMessage($string)
     {
+        @trigger_error($string);
```

Comment:
this will trigger them as ``E_USER_ERROR``. IMO, it is too high. What about ``E_USER_NOTICE`` instead ? Lower than notice would be even better as this is a debugging info, but trigger_error does not have something lower (as it is not meant to be used as a debug logger)

btw, I'm not even convinced this is a good idea (at least not today as people will complain about silenced notices in the profiler)